core/src/main/scala/org/locationtech/rasterframes/expressions/package.scala

```diff
@@ -128,6 +128,9 @@ package object expressions {
 
     registry.registerExpression[DebugRender.RenderAscii]("rf_render_ascii")
     registry.registerExpression[DebugRender.RenderMatrix]("rf_render_matrix")
+    registry.registerExpression[RenderPNG.RenderCompositePNG]("rf_render_png")
```

Comment:
So this guy only takes one param but it is documented (in reference.pymd) as if it takes 3. Not sure what the intent is. I would think we should have one overload for composite PNG and another overload for ColorRamp PNG.

Reply:
#313